docs/source/notebooks/power_analysis.ipynb

```diff
@@ -0,0 +1,1412 @@
+{
```

Comment:
<p>Can you delete this cell. This is good practice, used in the pymc-examples repo for example, but at the moment we've not added that to all the notebooks. We can create a separate issue about that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/277/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
Done!